src/plugin.php

```diff
@@ -89,6 +89,7 @@ private function load_integrations() {
 		$this->integrations[] = new Domain_Dropdown( $option );
 		$this->integrations[] = new Inline_Script( $option );
 		$this->integrations[] = new Admin_Debug_Info( $option );
+		$this->integrations[] = new Indexing_Reason();
```

Comment:
can we use dependency injection here?

Reply:
No. That needs some setup. Probably worth a separate issue.